src/DotNetty.Transport/Bootstrapping/AbstractBootstrap.cs

```diff
@@ -253,7 +253,7 @@ public Task<IChannel> BindAsync(EndPoint localAddress)
             {
                 if (channel.Registered)
                 {
-                    channel.CloseAsync();
+                    await channel.CloseAsync();
```

Comment:
if this throws, it would mask an original error. It's better to have a try-catch around it and log using Logger (see `ServerBootstrap.Logger` for reference).